app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("app/cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) "
+            f" VALUES (?, ?)",
+            (first_name, last_name)
+        )
+
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actors = self._connection.execute(
```

Comment:
Consider adding error handling for database operations to manage exceptions and improve the robustness of the code.